python_modules/dagster/dagster_tests/core_tests/instance_tests/test_instance.py

```diff
@@ -532,7 +532,7 @@ def test_dagster_home_not_set():
 
 def test_invalid_configurable_class():
```

Comment:
should there be a new test case here that verifies the new behavior added in this PR?

I guess its tricky to simulate the case where certain dagster-xxx modules are not installed since they will typically be installed in the tox suite

Reply:
Added some new test cases that rely on mocking. Can't load from the real dagster-plus module yet because it hasn't been published with the cloud instance

Reply:
Switched the test cases to import test modules of dagster-cloud and dagster-plus